packages/expo-font/src/Font.ts

```diff
@@ -29,9 +36,19 @@ export function processFontFamily(fontFamily: string | null): string | null {
  */
 export function isLoaded(fontFamily: string): boolean {
   if (Platform.OS === 'web') {
-    return fontFamily in loaded || !!ExpoFontLoader.isLoaded(fontFamily);
+    return isLoadedInCache(fontFamily) || !!ExpoFontLoader.isLoaded(fontFamily);
   }
-  return fontFamily in loaded || ExpoFontLoader.customNativeFonts?.includes(fontFamily);
+  return isLoadedNative(fontFamily);
+}
+
+/**
+ * Synchronously get all the fonts that have been loaded.
+ * This includes fonts that were bundled at build time using the config plugin, as well as those loaded at runtime using `loadAsync`.
+ *
+ * @returns Returns array of font family names that have been loaded.
+ */
+export function getLoadedFonts(): string[] {
+  return ExpoFontLoader.loadedFonts;
```

Comment:
We should avoid having differences between the native and JS APIs like this (property vs function). Since we already agreed on exposing it as `getLoadedFonts` function, let's make it so in the module definitions too.

Reply:
done